assets/js/modules/adsense/components/dashboard/AdSenseDashboardWidget.js

```diff
@@ -130,7 +130,7 @@ class AdSenseDashboardWidget extends Component {
 
 		// Hide AdSense data display when we don't have data.
 		const wrapperClass = ( loading || ! receivingData || zeroData ) ? 'googlesitekit-nodata' : '';
-		const currentDateRange = getCurrentDateRange( dateRange );
+		const currentDateRange = getCurrentDateRangeDayCount( dateRange );
```

Comment:
Maybe worth renaming this variable to `currentDayCount` or `currentNumOfDays` or something like that. It's no longer a date range at this point.